x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/prebuilt_rule_customization/customization_enabled/rule_customization.ts

```diff
@@ -0,0 +1,909 @@
+          expect(body.rule_source.type).toEqual('external');
+        });
+
+        it('max_signals field', async () => {
+          const { body } = await securitySolutionApi
+            .patchRule({
+              body: {
+                rule_id: 'rule_1',
+                max_signals: 42,
+              },
+            })
+            .expect(200);
+
+          expect(body.rule_source.is_customized).toEqual(true);
+          expect(body.rule_source.type).toEqual('external');
+        });
+
+        it('investigation_fields field', async () => {
+          const { body } = await securitySolutionApi
+            .patchRule({
+              body: { rule_id: 'rule_1', investigation_fields: { field_names: ['blob', 'boop'] } },
+            })
+            .expect(200);
+
+          expect(body.rule_source.is_customized).toEqual(true);
+          expect(body.rule_source.type).toEqual('external');
+        });
+
+        it('rule_name_override field', async () => {
+          const { body } = await securitySolutionApi
+            .patchRule({
+              body: { rule_id: 'rule_1', rule_name_override: 'override string' },
+            })
+            .expect(200);
+
+          expect(body.rule_source.is_customized).toEqual(true);
+          expect(body.rule_source.type).toEqual('external');
+        });
+
+        it('timestamp_override field', async () => {
+          const { body } = await securitySolutionApi
+            .patchRule({
+              body: { rule_id: 'rule_1', timestamp_override: 'event.ingested' },
+            })
+            .expect(200);
+
+          expect(body.rule_source.is_customized).toEqual(true);
+          expect(body.rule_source.type).toEqual('external');
+        });
+
+        it('timeline_template fields', async () => {
+          const { body } = await securitySolutionApi
+            .patchRule({
+              body: { rule_id: 'rule_1', timeline_id: '123', timeline_title: 'timeline title' },
+            })
+            .expect(200);
+
+          expect(body.rule_source.is_customized).toEqual(true);
+          expect(body.rule_source.type).toEqual('external');
+        });
+
+        it('building_block_type field', async () => {
+          const { body } = await securitySolutionApi
+            .patchRule({
+              body: { rule_id: 'rule_1', building_block_type: 'building block string' },
+            })
+            .expect(200);
+
+          expect(body.rule_source.is_customized).toEqual(true);
+          expect(body.rule_source.type).toEqual('external');
+        });
+      });
+
+      describe('query rule fields', () => {
+        beforeEach(async () => {
+          await createPrebuiltRuleAssetSavedObjects(es, [ruleAsset]);
+          await installPrebuiltRules(es, supertest);
+        });
+
+        afterEach(async () => {
+          await deleteAllRules(supertest, log);
+          await deleteAllPrebuiltRuleAssets(es, log);
+        });
```

Comment:
Why we keep deleting and creating prebuilt rule assets in every test? Rule assets are read-only objects and we could pre-create all the assets needed for all the tests in this file, and do it once.